acceptance/tests/direct_puppet/supports_utf8.rb

```diff
@@ -7,24 +7,18 @@
       'audit:refactor',  # Review for agent side UTF validation.
       'server'
 
-
   app_type = File.basename(__FILE__, '.*')
   tmp_environment   = mk_tmp_environment_with_teardown(master, app_type)
 
   tmp_file = {}
   agents.each do |agent|
-    if agent.platform =~ /^(eos-4-i386|cumulus-2\.5|osx|sles-10)/
-      # skip_test doesn't work in with_puppet_running_on blocks (tableflip)
-      #   so we can't easily use expect_failure here
-      skip_test 'PUP-6217'
-    end
```

Comment:
PUP-6217 is still open, does that mean the code now works correctly on these platforms and we can close that other ticket? Or does the test no longer trigger bad behavior?

Could you trigger a jenkins run on this PR on the above platforms to make sure the test passes (if you haven't manually tested already)?

Reply:
The test works correctly on all the platforms pooler and hardware platforms. I ran the complete set before I put up the PR.

I'll take a look at PUP-6217